connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -420,6 +423,15 @@ message ExecutePlanResponse {
     // the execution is complete. If the server sends onComplete without sending a ResultComplete,
     // it means that there is more, and the client should use ReattachExecute RPC to continue.
   }
+
+  // This message is used to communicate progress about the query progress during the execution.
+  message ExecutionProgress {
+    int64 num_tasks = 1;
+    int64 num_completed_tasks = 2;
+    int64 num_stages = 3;
+    int64 num_completed_stages = 4;
```

Comment:
We're carrying the state and task information but I can't see in the PR description example. How are they displayed?